package.json

```diff
@@ -57,5 +57,8 @@
   ],
   "resolutions": {
     "prosemirror-model": "1.11.2"
+  },
+  "volta": {
+    "node": "14.20.0"
```

Comment:
Are there any differences in using volta or not using it?

Reply:
！
That's a nodejs management tool. Kinda like the nvm.
This config should be removed.